test/e2e/network/netpol/kubemanager.go

```diff
@@ -63,12 +63,20 @@ func (k *kubeManager) initializeCluster(model *Model) error {
 		for _, pod := range ns.Pods {
 			framework.Logf("creating/updating pod %s/%s", ns.Name, pod.Name)
 
-			kubePod, err := k.createPod(pod.KubePod())
+			thePod := pod.KubePod()
+			if framework.NodeOSDistroIs("windows") {
```

Comment:
This is another piece of logic that really belongs in the model.  Spreading this logic out all over the tests instead of in `Model` makes it really hard to understand what the tests are going to do.

Reply:
ya fixed resolvingggg